tests/state.data.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2015 Digital Bazaar, Inc. All rights reserved.
+ */
+ /* jshint node: true */
+
+'use strict';
+
+var helpers = require('./helpers');
+
+var data = {};
+module.exports = data;
+
+data.batchMessageStateData = [
+  {
+    msgData: {state: 'pending'}, batchData: {dirty: false},
+    result: {msgState: 'ready', dirty: false}
+  },
+  {
+    msgData: {state: 'pending'}, batchData: {dirty: true},
+    result: {msgState: 'ready', dirty: false}
+  },
+  {
+    msgData: {state: 'ready'}, batchData: {dirty: true},
+    result: {msgState: 'ready', dirty: true}
+  }
+];
+
+data.getUnbatchedMessageStateData = [
+	{msgData: {state: 'ready'}, batchData: {dirty: true, id: 1}, result: true},
```

Comment:
Change tabs to spaces in getUnbatchedMessageStateData and resetMessageStateData.